### DISCUSSION TITLE
High memory usage

### DISCUSSION THREAD

**Discussion by RockyMM** [zed-industries/zed#17141] [Feature Requests]
Hi all,

I left Zed opened for several days with a Java project of several hundred files, maybe early thousands.

Just a moment ago I had to force-quit Zed as this had happened:

![image](https://github.com/user-attachments/assets/58d916f6-c8f4-4400-a7ee-caa41e1cb5a3)

Has anyone else experienced high memory usage from Zed? Which diagnostic info I could gather the next time if someone is interested?

Thanks!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
